### ISSUE TITLE
Maintainers wanted

### ISSUE THREAD

**Issue by PodaruDragos** [remojansen/InversifyJS#1426]
This is a _call to arms_ for anyone who might be interested in helping `inversify` and `inversify-packages` get up to date and bug-free. We are currently overwhelmed and limited to adding contributions, but that might change if people will like to contribute.

Fell free to just respond in this issue if you are interested to contribute to this great ecosistem.

We now also  have a [discord](https://discord.com/invite/jXcMagAPnm) if you want to tag alone.

**Comment by glanna** [comment]
Hi, I'm interested in contributing to this project!  I created an [issue](https://github.com/inversify/inversify-binding-decorators/issues/207) to add support to request scope in the inversify-bind-decorators and I have a PR to merge. 

I'd be happy to be a contributor to this project.

**Comment by AGrzes** [comment]
I'm using `inversify` in several of my projects and I will try to find some time to contribute. Is there any area to focus on or simply select open issues and try to resolve them?

**Comment by PodaruDragos** [comment]
> I'm using `inversify` in several of my projects and I will try to find some time to contribute. Is there any area to focus on or simply select open issues and try to resolve them?

Hello, you can just pick w/e issues you would like to solve. Thanks.

**Comment by acald-creator** [comment]
I'm interested in contributing but I wanted to let you know that your Discord invite is no longer invalid. Do you have another Discord invite link that can be used?

**Comment by PodaruDragos** [comment]
> Discord invite link that can be used?

I have updated the invite link.

**Comment by theodesp** [comment]
@PodaruDragos I can also provide my spare time to maintain this project. I've done some contributions in the past as well. Let me know and I can start working on some tickets.

**Comment by PodaruDragos** [comment]
@theodesp sure, nice to have you on board

**Comment by arthurgavazza** [comment]
@PodaruDragos I'm new to open source, but I've been using Inversify in many projects at my current job and I'm very interested in becoming a contributor

**Comment by PodaruDragos** [comment]
hey @arthurgavazza, happy to have you onboard, please join our discord server so we can chat about it.

**Comment by egmacke** [comment]
@PodaruDragos I've been using inversify on several projects over the last few years and have started contributing to some of the ecosystem.

At least one of my contributions was critical for our use, it's been merged into inversify-express-utils, but there hasn't been a new release since 2021 (so we're currently using a forked version). Is there a plan for pushing more releases any time soon? I'm happy to help out if I can unblock anything that's holding this up.

**Comment by Jameskmonger** [comment]
@egmacke I tried to get publish access to that for a while, looks like I missed it being granted 😄 I will investigate the release process tonight.

**Comment by PodaruDragos** [comment]
> @PodaruDragos I've been using inversify on several projects over the last few years and have started contributing to some of the ecosystem.
> 
> At least one of my contributions was critical for our use, it's been merged into inversify-express-utils, but there hasn't been a new release since 2021 (so we're currently using a forked version). Is there a plan for pushing more releases any time soon? I'm happy to help out if I can unblock anything that's holding this up.

as far as I am aware only @dcavanagh has publish access. Dan friendly ping

**Comment by Jameskmonger** [comment]
I have access @PodaruDragos. It has `publish-please` so should be simple enough

**Comment by egmacke** [comment]
@Jameskmonger any update on getting new releases out? (I don't want to pollute this thread but not sure where else to ask about this)

Let me know if there's anything I can do to help.

**Comment by nabby27** [comment]
Have you considered using a rewards system to encourage developers to contribute to inversify? I recently released my new project [Opire](https://opire.dev) which does just this.

With Opire not only code owners can create rewards, rewards can be created by anyone (and can be accumulated for the same issue). This allows people to contribute not only with their knowledge and time but also with money. Additionally, the organization receives a percentage if the rewards are created by people outside the organization.

You can see the documentation at [docs.opire.dev](https://docs.opire.dev). If you have questions or want more information, you can contact me 👍

**Comment by PodaruDragos** [comment]
Hey everyone !! 
We now have a new admin @notaphplover. He was involved in the project in the past and now he has taken the lead on this project. For anyone interested in collaborating please feel free to reach out to me or him. 

I propose we keep this issue open to attract more people if anyone is interested.

**Comment by IbrahimShamma99** [comment]
Hello @PodaruDragos 
Do you still accept new maintainers?

**Comment by PodaruDragos** [comment]
hey @IbrahimShamma99 of course. Join the discord and get in touch with @notaphplover since I am no longer involved in `inversify` at all.

**Comment by amandeavor** [comment]
Hi @notaphplover — I’m interested in earning a focused maintenance role through contribution. My relevant background is TypeScript/Node tooling, regression-tested fixes, issue reproduction, dependency work, and CI/build troubleshooting.

I can start with a bounded lane in `inversify` or `inversify-packages`: reproduce current TypeScript/runtime compatibility reports, add focused regression tests, review small PRs against existing conventions, and help reduce stale issue/PR backlog before requesting permissions. My recent public TypeScript work includes tested contributions to `opencode-mem` (#279) and `svg2pdf.js` (#372).

**Comment by notaphplover** [comment]
Hey @amandeavor, thanks for your interest. Sure we can talk about it, feel free to join the discord channel and reach me (korosempai)
